### ISSUE TITLE
Numeric faceting does not return min, max, avg, etc stats in 0.26.0.rc

### ISSUE THREAD

**Issue by iamarora** [typesense/typesense#1257]
## Description

min, max range from the RangeFilter from InstantSearch always returns 0 for both.
This is happening in `0.26.0.rc16` it worked as expected in `0.26.0.rc1`

## Steps to reproduce
Sample schema
```
{
  "name": "test_int64",
  "fields": [
    {
      "name": "_id",
      "type": "string",
      "facet": false,
      "optional": true,
      "index": true,
      "sort": false,
      "infix": false,
      "locale": ""
    },
    {
      "name": "timestamp_int64",
      "type": "int64",
      "facet": true,
      "optional": true,
      "index": true,
      "sort": false,
      "infix": false,
      "locale": ""
    }
  ],
  "default_sorting_field": "",
  "enable_nested_fields": false,
  "symbols_to_index": [],
  "token_separators": []
}
```
Sample data
```
[
  {
    "_id": "1",
    "timestamp_int64": 0
  },
  {
    "_id": "2",
    "timestamp_int64": 1695306256
  }
]
```

## Expected Behavior

The min, max values should return as per the int64 data.

## Actual Behavior

min, max is always returning 0.

## Metadata

**Typesense Version**: 0.26.0.rc16

**OS**: Ubuntu

**Comment by kishorenc** [comment]
@iamarora 

This is fixed in `0.26.0.rc21` . Can you please check? I have verified that the example that Jason has provided above now works.

**Comment by iamarora** [comment]
> @iamarora
> 
> This is fixed in `0.26.0.rc21` . Can you please check? I have verified that the example that Jason has provided above now works.

Yes, will check. Not able to download this - `curl https://dl.typesense.org/releases/0.26.0.rc16/typesense-server-0.26.0.rc21-amd64.deb` Has it been published?

**Comment by kishorenc** [comment]
You've `0.26.0.rc16` in the URL :)

**Comment by iamarora** [comment]
> You've `0.26.0.rc16` in the URL :)

Haha, thank you, my eagerness to test, messed up. Will report back. Thanks!

**Comment by iamarora** [comment]
Thanks all! This has been fixed in `0.26.0.rc22`
